shared/xccdf/system/software/gnome.xml

```diff
@@ -413,21 +413,21 @@ of the information system but does not want to logout because of the temporary n
 <title>Set GNOME3 Screensaver Lock Delay After Activation Period</title>
 <description>
 To activate the locking delay of the screensaver in the GNOME3 desktop when 
-the screensaver is activated, add or set <tt>lock-delay</tt> to <tt>uint32 0</tt> in
+the screensaver is activated, add or set <tt>lock-delay</tt> to <tt>uint32 5</tt> in
```

Comment:
@redhatrises totally agree. baby steps as @tbrunell gets up to speed on ssg/coding/scap :)